package.json

```diff
@@ -5,7 +5,8 @@
   "license": "apache-2.0",
   "scripts": {
     "start": "webpack-dev-server --host 0.0.0.0 --hot",
-    "build": "webpack -p",
+    "typecss": "find src -name '*css' -exec tcm -c -p {} \\;",
```

Comment:
Is `find` necessary here? https://github.com/Quramy/typed-css-modules suggests it isn't needed.

Reply:
_tcm -c -p src/**/*css_ doesn't work
![With Regex](https://i.ibb.co/QcrRC7r/tcm-regex.png "With Regex")
![Single file](https://i.ibb.co/dj1y32j/tcm-single.png "Single file")